website/www/site/layouts/_default/baseof.html

```diff
@@ -19,13 +19,13 @@
   {{ partial "header.html" . }}
   <div class="body__contained">
     {{ block "hero-section" . }}{{ end }}
+    {{ block "ctas-section" . }}{{ end }}
     {{ block "pillars-section" . }}{{ end }}
     {{ block "graphic-section" . }}{{ end }}
     {{ block "calendar-section" . }}{{ end }}
     {{ block "quotes-section" . }}{{ end }}
```

Comment:
Calendar and quotes sections behave a bit weirdly on smaller screen
![image](https://user-images.githubusercontent.com/15073128/103221346-b0de3380-4922-11eb-953b-4efff7e890dd.png)

Reply:
I just fixed calendar (there was missing class implementation), about the quotes it was done in previous pr's